hikari/interactions/base_interactions.py

```diff
@@ -183,6 +183,15 @@ class PartialInteraction(snowflakes.Unique, webhooks.ExecutableWebhook):
     version: int = attr.field(eq=False, repr=True)
     """Version of the interaction system this interaction is under."""
 
+    locale: str = attr.field(eq=False, repr=True)
```

Comment:
While the lib doesn't use PartialInteraction like this in the standard impl right now, PartialInteraction is meant to be able to support Ping interactions so these should probably be added to each sub-class instead of the base class

Reply:
Should be resolved in this commit https://github.com/hikari-py/hikari/pull/967/commits/16dd451e99d1354d97be3245b4f454b2587a4894